templates/etc_init_archivematica-mcp-server.conf.j2

```diff
@@ -25,13 +25,15 @@ pre-start script
     done
     gearadmin --getpid || exit 1
 
+    {% if archivematica_src_am_database_host == "localhost" %}
     # Wait for mysql to start - timeout eventually
     for i in $(seq 1 10)
     do
         mysqladmin ping && break
         sleep 3
     done
     mysqladmin ping || exit 1
+    {% endif %}
```

Comment:
MCPServer and MCPClient needs to implement retries. This should be happening soon for RDSS.